DemoServer/AdditionalLanguages/nodejs/demo/staticIndexes/projectIndexResults/metadata.json

```diff
@@ -0,0 +1,177 @@
+                    "Type": "link",
+                    "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#static-indexes-and-other-advanced-options"
+                },
+                {
+                    "Title": "Working with Indexes",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#working-with-indexes"
+                }
+            ]
+        },
+        {
+            "Title": "Deploy the index",
+            "Slug": "step-3",
+            "DescriptionLines": [
+                "* Use the `execute()` method to deploy the index to the server.",
+                "",
+                "* Once the index is deployed, the server will start the _'indexing process'_,",
+                "  in which the index iterates over the documents, building a map between the terms derived from the document fields indexed and the actual documents that contain them.<br>",
+                "",
+                "* In the case of a Map-Reduce index, a second stage is applied where the mapped entries are reduced to some aggregated value.",
+                "",
+                "* After the first indexing run, the index will keep this map current without re-processing the entire data,",
+                "  and will only update the relevant details upon any update/change to any document related to the index."
+            ],
+            "Assets": [
+                //TODO
+                //{
+                //    "Title": "Sending Indexes to Server",
+                //    "Type" : "link",
+                //    "Url"  : "https://ravendb.net/docs/article-page/latest/nodejs/indexes/creating-and-deploying#sending-to-server"
+                //},
+                {
+                    "Title": "Deploying Indexes",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#deploying-indexes"
+                },
+                {
+                    "Title": "Indexing Process",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/nodejs/studio/database/indexes/indexes-overview#indexing-process"
+                }
+            ]
+        },
+        {
+            "Title": "Query for documents",
+            "Slug": "step-4",
+            "DescriptionLines": [
+                "* Query the 'Employees' collection using the defined index.",
+                "",
+                "* The resulting documents will include only employees that started working after 'startYear'."
+            ],
+            "Assets": [
+            ]
+        },
+        {
+            "Title": "Set the query",
+            "Slug": "step-5",
+            "DescriptionLines": [
+                "* Pass the raw-query-string to the session `rawQuery()` method."
```

Comment:
=>
* Define the query, pass the raw-query-string to the session `rawQuery()` method.